ods_ci/tasks/Resources/RHODS_OLM/uninstall/uninstall.robot

```diff
@@ -70,10 +70,10 @@ Uninstall RHODS V2
     ...    oc delete dscinitialization $(oc get dscinitialization --no-headers | awk '{print $1}')
     Should Be Equal As Integers	${return_code}	 0   msg=Error deleting DSCInitialization CR
     ${return_code}    ${output}    Run And Return Rc And Output
-    ...    oc delete subscription $(oc get subscription -n redhat-ods-operator --no-headers | awk '{print $1}')
+    ...    oc delete subscription $(oc get subscription -n redhat-ods-operator --no-headers | awk '{print $1}') -n redhat-ods-operator
```

Comment:
we should start using the global variable containing the namespace  - added by Jorge recently

Reply:
sure ..we can do that as well